internal/cmd/modules/modules.go

```diff
@@ -135,6 +151,43 @@ func listManagedModules(cfg *modulesConfig) ([]string, clierror.Error) {
 	return moduleNames, nil
 }
 
+func listInstalledModules(cfg *modulesConfig) ([]string, clierror.Error) {
+	resp, err := http.Get("https://raw.githubusercontent.com/kyma-project/community-modules/main/model.json")
+	if err != nil {
+		return nil, clierror.Wrap(err, clierror.New("while getting modules list from github"))
+	}
+	defer resp.Body.Close()
+
+	var template model.Module
+
+	template, respErr := handleResponse(err, resp, template)
+	if respErr != nil {
+		return nil, clierror.WrapE(respErr, clierror.New("while handling response"))
+	}
+
+	var out []string
+	for _, rec := range template {
+		short := strings.Split(rec.Versions[0].ManagerPath, "/")
+		version := rec.Versions[0].Version
+		deployment, err := cfg.KubeClient.Static().AppsV1().Deployments("kyma-system").Get(cfg.Ctx, short[len(short)-1], metav1.GetOptions{})
```

Comment:
What is `short[len(short)-1]`? It's hard to understand. Could we name this object (i.e. `managerName` or something similar)?

Reply:
Applied suggested name.